.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: Build
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+      - name: Configure yarn caching
+        uses: actions/cache@v2
+        with:
+          path: ~/.yarn
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/workflows/prettier.yml') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+          cache: "yarn"
+      - name: Install dependencies
+        run: yarn install
+      - name: Format the code
+        run: yarn run prettier
+      - name: Check the code style
+        run: yarn run lint:eslint
+      - name: Check the types
+        run: yarn run check-types
+      - name: Build the project
+        run: yarn run build
```

Comment:
@andrehadianto on second thoughts, lets remove build here to reduce Github actions build time. Since we already build it on Cloudfare, this is not necessary.